src/main/g8/src/main/scala/com/lightbend/akka/http/sample/UserRoutes.scala

```diff
@@ -0,0 +1,99 @@
+package com.lightbend.akka.http.sample
+
+import akka.actor.{ ActorRef, ActorSystem }
+import akka.event.Logging
+
+import scala.concurrent.duration._
+import akka.stream.ActorMaterializer
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.Http.ServerBinding
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.{ ExceptionHandler, Route }
+import akka.http.scaladsl.server.directives.MethodDirectives.delete
+import akka.http.scaladsl.server.directives.MethodDirectives.get
+import akka.http.scaladsl.server.directives.MethodDirectives.post
+import akka.http.scaladsl.server.directives.RouteDirectives.complete
+import akka.http.scaladsl.server.directives.PathDirectives.path
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+import scala.util.{ Failure, Success }
+import com.lightbend.akka.http.sample.UserRegistryActor._
+import akka.pattern.ask
+import akka.util.Timeout
+
+//#user-routes-class
+trait UserRoutes extends JsonSupport {
+  //#user-routes-class
+
+  // we leave these abstract, since they will be provided by the App
+  implicit def system: ActorSystem
+
+  lazy val log = Logging(system, classOf[UserRoutes])
+
+  // other dependencies that UserRoutes use
+  def userRegistryActor: ActorRef
+
+  // Required by the `ask` (?) method below
+  implicit lazy val timeout = Timeout(5.seconds) // usually we'd obtain the timeout from the system's configuration
+
+  //#all-routes
+  //#users-get-post
+  //#users-get-delete   
+  lazy val userRoutes =
+    pathPrefix("users") {
+      concat(
+        //#users-get-delete
+        pathEnd {
+          concat(
+            get {
+              val users: Future[Users] = (userRegistryActor ? GetUsers).mapTo[Users]
```

Comment:
With a line break after `=` (also for the ask below) I think we could avoid getting a horizontal scroll bar on the sample code in the rendered output.